code/modules/mining/aux_base.dm

```diff
@@ -224,7 +224,7 @@ MAPPING_DIRECTIONAL_HELPERS(/obj/machinery/computer/auxiliary_base, 32)
 			var/turf/place = colony_turfs[i]
 			if(!place)
 				return BAD_COORDS
-			if(!istype(place.loc, /area/lavaland/surface))
+			if(!(istype(place.loc, /area/lavaland/surface) || istype(place.loc, /area/icemoon)))
```

Comment:
```suggestion
			if(!istype(place.loc, /area/lavaland/surface) || !istype(place.loc, /area/icemoon))
```

Reply:
Are you sure i should disable landing on the surface itself? It would probably need its own error message.

I do agree about breaking up the parentheses to make it more readable (also need to flip the || to &&).

Reply:
I have broken up the parentheses. Are you sure we still shouldn't let the base land on the surface? It has mining enabled in the map json.

Reply:
Surface is the top level of Icemoon, where most of the station is? There's nothing to explore there, or fauna to fight, so I don't think theres much use for it there, so it would be lame if you random dropped it and landed 2 feet away from the station.

Reply:
I will figure out a proper feedback message for it then with a separate check.